### PR TITLE
+ Saul 

### PR DESCRIPTION
**Pull request by danyaljj**
... will go public tonight.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1121,6 +1122,7 @@ on MNIST digits[DEEP LEARNING]
 * [H2O Sparkling Water](https://github.com/h2oai/sparkling-water) - H2O and Spark interoperability.
 * [FlinkML in Apache Flink](https://ci.apache.org/projects/flink/flink-docs-master/apis/batch/libs/ml/index.html) - Distributed machine learning library in Flink
 * [DynaML](https://github.com/mandar2812/DynaML) - Scala Library/REPL for Machine Learning Research
+* [Saul](https://github.com/IllinoisCogComp/saul/) - Flexible Declarative Learning-Based Programming.
```
Comment (robsonpeixoto) [inline README.md]: broken link

Comment (robsonpeixoto) [inline README.md]: or private repo

Comment (danyaljj) [inline README.md]: check now.

**Comment by josephmisiti** [comment]
thanks

**Pull request merged**